contracts/TokenUse.sol

```diff
@@ -190,6 +190,21 @@ contract TokenUse is DSAuth, ITokenUse, SettingIds {
         }
     }
 
+    function registerTokenStatus(uint256 _tokenId, address _owner, address _user, uint256 _startTime, uint256 _endTime, uint256 _price, address _acceptedActivity) public auth {
```

Comment:
what is the user here? the siring contracts?

Reply:
_user is the bidder

Reply:
Here, there is a new token status created, buy no token transferred in, it is not compliant with existing rules.